templates/zerver/dev_tools.html

```diff
@@ -73,6 +73,11 @@ <h1>Useful development URLs</h1>
                     <td>None needed</td>
                     <td>Invalid confirmation link page</td>
                 </tr>
+                <tr>
+                    <td><a href="/devtools/integrations">/devtools/integrations</a></td>
+                    <td>Replaces the need for <code>./manage.py send_webhook_fixture_message</code></td>
```

Comment:
I would replace this one with just the command and remove the `Replacds the need for" part.